vendor/github.com/aws/aws-sdk-go/aws/awsutil/prettify.go

```diff
@@ -1,113 +0,0 @@
-package awsutil
```

Comment:
Bottom line is if you want it to be vendored with no usage, I'm ok with that. I was only trying to limit the debug output.